pkg/conf/conf.go

```diff
@@ -92,6 +92,24 @@ func LoadHarvestConfig(configPath string) error {
 	return nil
 }
 
+func SafeHarvestConfig(configPath string) error {
+	var (
+		contents []byte
+		err      error
+		fileName string
+	)
+	if !configRead {
+		panic("config not read")
+	}
+	if contents, err = yaml.Marshal(&Config); err != nil {
+		return err
+	}
+	if filepaths := strings.Split(configPath, "/"); len(filepaths) > 1 {
```

Comment:
config path can have more than 2 slashes. In that case filename would be wrong?

Reply:
good catch @rahulguptajss - better to use path.Base or path.Split